Dockerfile

```diff
@@ -92,4 +92,5 @@ ENV LANG C.UTF-8
 ENV TEST_DEBUG=0
 
 WORKDIR /root/lightning-integration
+COPY Makefile /root/lightning-integration/Makefile
```

Comment:
Yeah, I think it's better to take things from your local lightning-integration rather than cloning it again. The main problem here, I think, is making changes locally and see nothing happening.
I will do that instead and do a separate PR with only that since this one seems problematic for other reasons.